verify-service/src/main/java/io/inji/verify/dto/authorizationRequest/StatusDto.java

```diff
@@ -0,0 +1,15 @@
+package io.inji.verify.dto.authorizationRequest;
+
+import io.inji.verify.enums.SubmissionState;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class StatusDto {
+    SubmissionState status;
```

Comment:
/{requestId}/status - SubmissionState status; - is little confusing since the request is expecting a request status and the status dto has a enum called SubmissionState and the field name is status.

Reply:
Lets just send the status alone as a response then. That makes more sense right.

Reply:
As discusssed, lets have separate enums from VPRequestStatus and VPResultStatus and lets construct a proper json response only with status field when responding from vp request status API.

Reply:
I think we discussed about removing COMPLETED and change to something  like VP_SUBMITTED / VP_RECEIVED